<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mri_imag_Widget_1</class>
 <widget class="QWidget" name="mri_imag_Widget_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imag</string>
  </property>
  <widget class="QWidget" name="plotWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>600</width>
     <height>700</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="plotLayout"/>
  </widget>
  <widget class="QWidget" name="controlWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>350</width>
     <height>700</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QLabel" name="freqLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency/MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="freqValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>95</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>60.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>15.670000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectionLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>112</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Projection Axis, x/y/z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acquireButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Acquire</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProjLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3D Imaging control panel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadShimButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>load shims</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroShimButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>610</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>zero shims</string>
    </property>
   </widget>
   <widget class="QLabel" name="shimLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Shim strength</string>
    </property>
   </widget>
   <widget class="QLabel" name="shimLabel_x">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="shimLabel_y">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="shimLabel_z">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="shimUnitLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shim_x">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>510</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="shim_y">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>510</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="shim_z">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>510</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matSizeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Matrix size</string>
    </property>
   </widget>
   <widget class="QLabel" name="seqTypeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sequence type</string>
    </property>
   </widget>
   <widget class="QLabel" name="read">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>240</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="pe1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PE1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="size1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FOVLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FOV/mm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FOV1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="FOV2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="seqType">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reconButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Recon</string>
    </property>
   </widget>
   <widget class="QComboBox" name="npe">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="npe2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pe2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PE2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FOV3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
